debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/naming/DefaultTableNamingStrategy.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc.naming;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import io.debezium.bindings.kafka.KafkaDebeziumSinkRecord;
+import io.debezium.connector.jdbc.JdbcSinkConnectorConfig;
+import io.debezium.sink.naming.DefaultCollectionNamingStrategy;
+
+/**
+ * Default implementation of the {@link TableNamingStrategy} where the table name is driven
+ * directly from the topic name, replacing any {@code dot} characters with {@code underscore}
+ * and source field in topic.
+ *
+ * @author Chris Cranford
+ */
+public class DefaultTableNamingStrategy extends DefaultCollectionNamingStrategy implements TableNamingStrategy {
```

Comment:
@jpechane @Naros I have added another commit to re-add the old API, mapping to the new implementation. Including test scenarios for using manually provided `DefaultTableNamingStrategy` as config option and make derived implementations work. Wdyt?